components/esp32/ld/esp32.common.ld

```diff
@@ -151,11 +151,13 @@ SECTIONS
     *(.rodata1)
     __XT_EXCEPTION_TABLE_ = ABSOLUTE(.);
     *(.xt_except_table)
-    *(.gcc_except_table)
+    *(.gcc_except_table.*)
```

Comment:
In case -ffunction-sections switch isn't used LSDAs will be in the .gcc_except_table, but it will not be matched by the ```*(.gcc_except_table.*)```.